internal/client/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+
+	"foxygo.at/pony/pb/echo"
+	"github.com/alecthomas/kong"
+	"google.golang.org/grpc"
+)
+
+var version = "v0.0.0"
+
+type config struct {
+	Address string           `help:"hostname:port" default:"localhost:9090"`
+	Message string           `arg:"" help:"message to send" default:"Hello 🌏"`
+	Version kong.VersionFlag `short:"V" help:"Print version information" group:"Other:"`
+
+	out io.Writer
+}
+
+func main() {
+	cfg := &config{out: os.Stdout}
+	ctx := kong.Parse(cfg, kong.Vars{"version": version})
```

Comment:
i have always called this `kctx`. `ctx` to me always means `context.Context`. what do you think?

same in server if you agree (and I assume dynamicserver too, but i'm not there yet).

Reply:
disagree - convention over being right (unless it's really bad), makes code easier to read. 
all kong code _I_ see floating around uses ctx. if it has to be different i'd prefer ktx.
anyway. not important enough. changed, cause i want you on my side.